core/types/block.go

```diff
@@ -171,7 +179,8 @@ func (h *Header) EmptyReceipts() bool {
 type Body struct {
 	Transactions []*Transaction
 	Uncles       []*Header
-	Withdrawals  []*Withdrawal `rlp:"optional"`
+	Withdrawals  []*Withdrawal     `rlp:"optional"`
+	Witness      *ExecutionWitness `rlp:"-"`
```

Comment:
Witness should not be a part of Body because the Body struct represents what's stored in the DB/sent over network etc. It should only be in types.Block.